config/env.ini

```diff
@@ -65,27 +65,31 @@ UPX_EXEC="${PKG_ROOT_PATH}\bin\upx.exe"
 SPC_MICRO_PATCHES=static_extensions_win32,cli_checks,disable_huge_page,vcruntime140,win32,zend_stream,cli_static
 
 [linux]
-; Linux can use different build toolchain, but the toolchain can not be changed in this file:
-; - musl (default): used for general linux distros, can build `musl-static` target only.
-; - zig (WIP): used for general linux distros, can build `musl` and `glibc` targets.
-; - musl-native: used for alpine linux, can build `musl-static` and `musl`(WIP) target.
-; - gnu-native (assume): used for general linux distros, can build `glibc` target only and have portability issues.
+; Linux can use different build toolchains.
+; - musl (default, when SPC_LIBC=musl): used for general linux distros, can build `musl` (statically linked) only.
+; - zig (will become default): usable on all Linux distros, can build `-musl`, `arch-linux-musl -dynamic` and `arch-linux-gnu` targets. Can specify version such as `x86_64-linux-gnu.2.17`.
+; - musl-native: used for alpine linux, can build `musl` and `musl -dynamic` target.
+; - gnu-native: used for general linux distros, can build gnu target for the installed glibc version only.
 
-; build target:
-; - musl-static (default): pure static linking, using musl-libc, can run on any linux distro.
-; - musl: static linking with dynamic linking to musl-libc, can run on musl-based linux distro.
-; - glibc: static linking with dynamic linking to glibc, can run on glibc-based linux distro.
-
-; include PATH for musl libc.
+; LEGACY option to specify the target, switch to SPC_TARGET with zig toolchain instead
 SPC_LIBC=musl
+
+; Recommended: specify your target here. Zig toolchain will be used.
+; examples:
+; `native-native-gnu`           - links against glibc, current OS version
+; `native-native-gnu.2.17`      - links against glibc, version 2.17
+; `native-native`               - static linking against musl
```

Comment:
`native-native` doesn't seem to work on my debian host. `native-native-musl` works fine.

Reply:
native-native should expand to the same as native-native-gnu on debian. It works in the CI and locally for me. I have to fix the incorrect description here, though.